src/webgpu/util/conversion.ts

```diff
@@ -962,27 +963,7 @@ const boolType = new ScalarType('bool', 4, false, (buf: Uint8Array, offset: numb
 /** Type is a ScalarType, VectorType, MatrixType or ArrayType. */
 export type Type = ScalarType | VectorType | MatrixType | ArrayType;
 
-/** Type holds pre-declared Types along with helper constructor functions. */
-export const Type = {
-  abstractInt: abstractIntType,
-  'abstract-int': abstractIntType,
-  i32: i32Type,
-  u32: u32Type,
-  i16: i16Type,
-  u16: u16Type,
-  i8: i8Type,
-  u8: u8Type,
-
-  abstractFloat: abstractFloatType,
-  'abstract-float': abstractFloatType,
-  f64: f64Type,
-  f32: f32Type,
-  f16: f16Type,
-
-  bool: boolType,
-
-  vec: (width: number, elementType: ScalarType) => VectorType.create(width, elementType),
-
+const kVecTypes = {
```

Comment:
nit: kAllVecTypes

Reply:
Similar to `kAllMatrices`, generated a `kAllVecTypes` rather than change this.